### PR TITLE
Improving error messaging around SSL issues, pointing to docs

### PR DESCRIPTION
**Pull request by iambmelt**
See notes in related [IcM (corpnet reqd)](https://portal.microsofticm.com/imp/v3/incidents/details/198246518/home)

Gist:
- Misconfigured ADFS SSL certs frequently cause issues for customers using on-prem federation
- This PR links off to guidance docs on how to resolve common issues

### CODE REVIEW

File: common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java [hunk 1]
```diff
@@ -136,18 +135,21 @@ public void onReceivedSslError(final WebView view,
         super.onReceivedSslError(view, handler, error);
         handler.cancel();
 
+        final String sslHelpUrl = "https://docs.microsoft.com/en-us/troubleshoot/azure/active-directory/adal-authenticate-android-devices-fail";
```
Comment (iambmelt) [inline common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java:138]: @hamiltonha Do we have guidance on including pointers to docs URLs in error messages? Do we need to create a `go.microsoft` or `aka.ms` URL for it?

Comment (hamiltonha) [reply common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java:138]: @iambmelt we need to create FWLinks, for future reference: https://review.docs.microsoft.com/en-us/help/contribute/contribute-link-to-articles-from-the-user-interface?branch=master

Comment (iambmelt) [reply common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java]: Thank you!

File: common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java [hunk 2]
```diff
@@ -136,18 +135,21 @@ public void onReceivedSslError(final WebView view,
         super.onReceivedSslError(view, handler, error);
         handler.cancel();
 
+        final String sslHelpUrl = "https://go.microsoft.com/fwlink/?linkid=2138180";
+        final String errMsg = String.format(
+                "Received SSL Error during request. Please see %s for more info.",
+                sslHelpUrl
+        );
```
Comment (AdamBJohnsonx) [inline common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java:142]: Can this whole thing just be a static constant?

Comment (AdamBJohnsonx) [reply common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java:142]: Also, using concatenation is far more efficient than String.format, which we should probably avoid whenever feasible.

**Pull request merged**